frontend/src/metabase/public/containers/PublicDashboard.jsx

```diff
@@ -178,6 +177,7 @@ class PublicDashboard extends Component {
         actionButtons={
           buttons.length > 0 && <div className="flex">{buttons}</div>
         }
+        header={<StyledDashboardTabs location={this.props.location} />}
```

Comment:
nit: We use a bunch different things to make the header. Would prefer a different name here, so we know exactly what we are receiving in this. 
```suggestion
        dashboardTabs={<StyledDashboardTabs location={this.props.location} />}
```

Reply:
Since this is a short-term and very specific fix. I agree we should make this clear what it's suppose to represent.